lib/osf-components/addon/components/node-metadata-manager/component.ts

```diff
@@ -75,11 +71,13 @@ export default class NodeMetadataManagerComponent extends Component<Args> {
     async getGuidMetadata() {
         if (this.node) {
             const guidRecord = await this.store.findRecord('guid', this.node.id, {
-                include: 'metadata',
+                include: 'custom_metadata',
                 resolve: false,
             });
             this.guidType = guidRecord.referentType;
             this.metadata = guidRecord.customMetadata as CustomItemMetadataRecordModel;
+            notifyPropertyChange(this, 'metadata');
```

Comment:
Is this needed? Seeing as `this.metadata` is tracked, isn't this redundant?

Reply:
`metadata` is an object, therefore changes to it [aren't automatically tracked](https://embermap.com/topics/tracking-arrays/tracked-built-ins).